docker-compose/is-analytics/README.md

```diff
@@ -0,0 +1,56 @@
+# WSO2 Identity Server with Analytics
+
+Runs a pre configured Identity Server container and Identity Server Analytics container.
+
+## Prerequisites
+
+ * [Docker](https://www.docker.com/get-docker) and [Docker Compose](https://docs.docker.com/compose/install/#install-compose) are required for running this Docker Compose template.
+
+## Quick Start Guide
+
+  1. Pull WSO2 Identity Server and Identity Server Analytics Docker image or build it:
+
+     * [Identity Server Dockerfile](../../dockerfiles/is/README.md)
+     * [Identity Server Analytics Dockerfile](../../dockerfiles/is-analytics/README.md)
+
+  2. Pull MySQL Docker images:
+     ```
+     docker pull mysql:5.7.19
+     ```
+
+  3. Download the latest Identity Server Docker resources release zip file from the [releases](https://github.com/wso2/docker-is/releases) page or clone this repository and switch to the latest tag.
+
+  4. Switch to the docker-compose/is-analytics folder:
+     ```
+     cd [docker-is]/docker-compose/is-analytics
+     ```
+
+  5. Download [MySQL Connector/J](https://downloads.mysql.com/archives/c-j/) v5.1.35 and copy its JAR file to the following path:
+     ```
+     cp path/to/mysql-connector-java-5.1.35/mysql-connector-java-5.1.35-bin.jar [docker-is]/docker-compose/is-analytics/identity-server/repository/components/lib/
```

Comment:
Please do not use old 5.1.35 version, it has security issues: https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2017-3523

Reply:
@deki - I've created issue #29 to update the JDBC driver version. Will fix this as a separate PR.